### PR TITLE
Add basic info

### PR DESCRIPTION
**Pull request by pipo02mix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,55 @@ Velero relies on [Kubernetes Custom Resources](https://kubernetes.io/docs/concep
...
+    name: aws
+    config:
+      region: eu-central-1 
+
+initContainers:
+ - name: velero-plugin-for-aws
+   image: "velero/velero-plugin-for-aws:v1.0.0"
+   volumeMounts:
+   - mountPath: "/target"
+     name: plugins
```
Comment (pipo02mix) [inline README.md:45]:
```suggestion
     name: plugins
     credentials:
       secretContents:
         cloud: ""
```

File: README.md [hunk 2]
```diff
@@ -11,6 +11,58 @@ Velero relies on [Kubernetes Custom Resources](https://kubernetes.io/docs/concep
...
+     credentials:
+       secretContents:
+         cloud: ""
```
Comment (pipo02mix) [inline README.md:48]:
```suggestion
credentials:
  secretContents:
    cloud: ""
```

File: README.md [hunk 3]
```diff
@@ -11,6 +11,58 @@ Velero relies on [Kubernetes Custom Resources](https://kubernetes.io/docs/concep
 
 The tool contains a command line tool (CLI) to manage the backups. For more information look into the [official docs](https://velero.io/docs/v1.3.2/). 
 
+## Requirements
+
+- You should deploy only 1 release of this chart per Kubernetes cluster as it can run backups for entire cluster.
+- You must create the S3 bucket or Azure Blob upfront. 
+- You must generated Cloud Provider credentials upfront.
```
Comment (glitchcrab) [inline README.md:18]: just a minor nitpick to align it with the bullet point above, no biggie though.

```
- You must create the Cloud Provider credentials upfront.
```

Comment (pipo02mix) [inline README.md:18]:
```suggestion
- You must create the Cloud Provider credentials upfront.
```

**Pull request merged**